bot/cogs/utils.py

```diff
@@ -257,6 +257,24 @@ def check(m: Message) -> bool:
         embed.description = best_match
         await ctx.send(embed=embed)
 
+    @command(aliases=("poll",))
+    @with_role(*MODERATION_ROLES)
+    async def vote(self, ctx: Context, title: str, *options: str) -> None:
+        """
+        Build a quick voting poll with matching reactions with the provided options.
+
+        A maximum of 20 options can be provided, as Discord supports a max of 20
+        reactions on a single message.
+        """
+        if len(options) > 20:
+            raise BadArgument("I can only handle 20 options!")
+
+        options = {chr(i): f"{chr(i)} - {v}" for i, v in enumerate(options, start=127462)}
```

Comment:
I think it's worth defining `127462` as a variable or some constant so it's clearer what it is. While it's not super esoteric, it isn't immediately obvious either what's going on here.

Reply:
But I like it like this! It's super clean! haha. Fine, I'll assign a variable.